lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
> It was double escaping from the first cut: https://github.com/mochajs/mocha/pull/179/files#diff-d80a115516f0cc2ec0b46a54c60077faR88

I don't see how it was double escaping there, since `escape` was only used in the `cdata` fn, which was given `test.err.stack` directly.

Still, not as important now. I'm more interested in the second bullet point.